Dockerfile

```diff
@@ -1,19 +1,46 @@
-FROM node:22-slim AS base
-ENV PNPM_HOME="/pnpm"
-ENV PATH="$PNPM_HOME:$PATH"
-RUN corepack enable
+FROM node:22.13.1-alpine3.20 AS base
+
 WORKDIR /app
+
 COPY package.json pnpm-lock.yaml ./
-RUN corepack prepare
 
-FROM base AS prod-deps
-RUN --mount=type=cache,id=pnpm,target=/pnpm/store pnpm install --prod --frozen-lockfile
+RUN npm install -g pnpm@10.20.0 && \
+    npm install -g modclean
```

Comment:
We might want to wait on this one; recently we found out modclean is insane old and not good. Next.js standalone build has similar logic, but 20x better.

When we understand it, we would apply that instead.
For now I would either recommend without modclean, or to wait.

Reply:
will check.